test/multi-table.wast

```diff
@@ -1,8 +1,8 @@
 (module
   (type $none_=>_none (func))
   (type $A (struct))
-  (global $g1 (ref null $none_=>_none) (ref.func $f))
-  (global $g2 i32 (i32.const 0))
+  (global $g1 (import "env" "f") (ref null $none_=>_none))
+  (global $g2 (import "env" "g") i32)
```

Comment:
Why can't these stay as they were? Their initializers are trivial.

Reply:
These globals are used in constant expressions in the element segment definitions below. I'll revert the test change and enable GC on this test instead.